packages/app/src/features/errors/ErrorBoundary.tsx

```diff
@@ -29,6 +29,14 @@ export class ErrorBoundary extends Component<ErrorBoundaryProps, ErrorBoundarySt
 
   render() {
```

Comment:
nit, optional: no nested ifs and grouped falback case.


```suggestion
render() {
  const { hasError } = this.state;
  const { children, fallback } = this.props;

  if (!hasError) {
    return children;
  }

  if (!fallback) {
    throw this.state.error;
  }

  return typeof fallback === 'function'
    ? fallback({ error: this.state.error })
    : fallback;
}
```